src/main/docker/RNApeg.sh

```diff
@@ -55,9 +56,7 @@ elif [[ ! -f $FASTA.fai ]]; then
     usage
     exit 1
 elif [[ ! -d "$OUTPUT_DIR" ]]; then
-    usage
-    >&2 echo "ERROR: Output directory '$OUTPUT_DIR' does not exist; need mountpoint?"
-    exit 1
+    mkdir -p $OUTPUT_DIR
```

Comment:
I probably wouldn't autocreate the directory. This is for running inside Docker, so it wouldn't make sense to create the directory if it is only in the container's filesystem.

Reply:
Normally, I would agree, but since the CGC pipeline appends things that we can not control, it is necessary.  TBH, this isnt really an anti-pattern, its what I would expect a piece of software to do.  The anti-pattern is having the docker mount capabilities do it for you.  Either way, we will need to be able to create the output directory on the fly, so what solution do you think would be preferable?